airbyte-integrations/connectors/destination-redshift/src/test-integration/kotlin/io/airbyte/integrations/destination/redshift/RedshiftS3StagingInsertDestinationAcceptanceTest.kt

```diff
@@ -0,0 +1,277 @@
+/*
+ * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+ */
+package io.airbyte.integrations.destination.redshift
+
+import com.amazon.redshift.util.RedshiftTimestamp
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.node.ObjectNode
+import io.airbyte.cdk.db.Database
+import io.airbyte.cdk.db.factory.ConnectionFactory.create
+import io.airbyte.cdk.db.factory.DatabaseDriver
+import io.airbyte.cdk.db.jdbc.JdbcUtils.DATABASE_KEY
+import io.airbyte.cdk.db.jdbc.JdbcUtils.HOST_KEY
+import io.airbyte.cdk.db.jdbc.JdbcUtils.PASSWORD_KEY
+import io.airbyte.cdk.db.jdbc.JdbcUtils.PORT_KEY
+import io.airbyte.cdk.db.jdbc.JdbcUtils.USERNAME_KEY
+import io.airbyte.cdk.integrations.base.JavaBaseConstants.COLUMN_NAME_DATA
+import io.airbyte.cdk.integrations.base.JavaBaseConstants.COLUMN_NAME_EMITTED_AT
+import io.airbyte.cdk.integrations.standardtest.destination.JdbcDestinationAcceptanceTest
+import io.airbyte.cdk.integrations.standardtest.destination.TestingNamespaces.generate
+import io.airbyte.cdk.integrations.standardtest.destination.TestingNamespaces.isOlderThan2Days
+import io.airbyte.cdk.integrations.standardtest.destination.comparator.TestDataComparator
+import io.airbyte.commons.io.IOs.readFile
+import io.airbyte.commons.json.Jsons
+import io.airbyte.commons.json.Jsons.deserialize
+import io.airbyte.commons.string.Strings.addRandomSuffix
+import io.airbyte.integrations.destination.redshift.RedshiftStagingS3Destination.Companion.SSL_JDBC_PARAMETERS
+import io.airbyte.integrations.destination.redshift.operations.RedshiftSqlOperations
+import org.jooq.DSLContext
+import org.jooq.Record
+import org.jooq.impl.DSL
+import org.junit.jupiter.api.Disabled
+import org.junit.jupiter.api.parallel.Execution
+import org.junit.jupiter.api.parallel.ExecutionMode
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import java.nio.file.Path
+import java.sql.Connection
+import java.sql.SQLException
+import java.time.ZoneOffset
+import java.time.ZonedDateTime
+import java.time.format.DateTimeFormatterBuilder
+import java.time.temporal.ChronoField
+import java.util.Optional
+import java.util.function.Function
+import java.util.stream.Collectors
+
+/**
+ * Integration test testing [RedshiftStagingS3Destination]. The default Redshift integration
+ * test credentials contain S3 credentials - this automatically causes COPY to be selected.
+ */
+// these tests are not yet thread-safe, unlike the DV2 tests.
+@Execution(ExecutionMode.SAME_THREAD)
+@Disabled
+class RedshiftS3StagingInsertDestinationAcceptanceTest : JdbcDestinationAcceptanceTest() {
+  // config from which to create / delete schemas.
+  private lateinit var baseConfig: JsonNode
+
+  // config which refers to the schema that the test is being run in.
+  // override the getter name, because the base class declares a getConfig method, which clashes.
+  // Eventually we should just replace the super method with a native kotlin `abstract val`.
+  @get:JvmName("getConfig_")
```

Comment:
@gisripa how do you feel about this :/ 

it's kind of gross, but (a) we know how to fix it, and (b) we want to kill the legacy DAT suites anyway, so... :shrug:?

I could also be convinced to rename the var, or just give up on this conversion entirely

Reply:
ah gross. yeah don't worry about it since it is disabled and also bare minimum changes to the auto-conversion (even if it means silencing warnings in this PR)

Reply:
this ends up not mattering b/c I want to delete the DAT classes anyway :P